src/Compilers/VisualBasic/Portable/BoundTree/BoundPropertyAccess.vb

```diff
@@ -23,10 +27,21 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 isLValue:=propertySymbol.ReturnsByRef,
                 receiverOpt:=receiverOpt,
                 arguments:=arguments,
+                defaultArguments:=defaultArguments,
                 type:=GetTypeFromAccessKind(propertySymbol, accessKind),
                 hasErrors:=hasErrors)
         End Sub
 
+
+        Public Sub New(syntax As SyntaxNode, propertySymbol As PropertySymbol, propertyGroupOpt As BoundPropertyGroup, accessKind As PropertyAccessKind, isWriteable As Boolean, receiverOpt As BoundExpression, arguments As ImmutableArray(Of BoundExpression), Optional hasErrors As Boolean = False)
+            Me.New(syntax, propertySymbol, propertyGroupOpt, accessKind, isWriteable, receiverOpt, arguments, defaultArguments:=BitVector.Null, hasErrors)
```

Comment:
>defaultArguments:=BitVector.Null [](start = 110, length = 32)

It is better to not have constructors with assumptions like that, they make it easy to drop important information when not intended.